projects/ui-components/lf-login/lf-login.component.ts

```diff
@@ -680,4 +695,32 @@ export class LfLoginComponent implements OnChanges, OnDestroy {
       return secondString;
     }
   }
+
+  private async beforeFetchRequestAsync(url: string, request: RequestInit): Promise<LfBeforeFetchResult> {
```

Comment:
mark these as @Internal (in a jsdoc) so that they are not exported into the types package